launcher/src/main/kotlin/com/cognifide/gradle/aem/launcher/ForkScaffolder.kt

```diff
@@ -3,10 +3,16 @@ package com.cognifide.gradle.aem.launcher
 class ForkScaffolder(private val launcher: Launcher) {
 
     fun scaffold() {
+        addGradleProperties()
         savePropertiesTemplate()
         savePropertiesDefinitions()
     }
 
+    private fun addGradleProperties() = launcher.workFile("gradle.properties") {
```

Comment:
because `workFileOnce` checks if the file exists before creating it, and if it does exist it returns without doing anything, so can't be used to edit files that are already there

Reply:
but this is intentional :) that existing files will have preference

Reply:
GAP launcher is usually run in projects without Gradle support at all

Reply:
But when I create a "gradle.properties" file in the "gradle/fork/" directory, it's doesn't work (as if I added nothing), that's why I edited the "gradle.properties" on the root folder and named the method with `add` instead of `save`.
am I messing something in here?